karma.conf.js

```diff
@@ -94,6 +109,8 @@ function combinedLoaders() {
     case 'istanbulInstrumenter':
     case 'tslint':
       return aggregate;
+    case 'ts':
```

Comment:
Here I tried to ignore the 'ts' loader (because it is using AoT while test is supposed to use JiT). So it seems I have to add a `case 'ts'` to do that?